index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+<body>
+  <header class="header">
+    <div class="toolbar logo">John Doe</div>
+  <nav class="toolbar"><img src="image/Union.png" class="imagcss" alt="dropdown menu"></nav></header>
+  <div class="headline">
+<h1 class="primary">Hey There.
+  I’m Ghilain</h1>
+  <h2 class="support">I’am a Software Developer</h2>
+  <p class="description">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+    don’t hestiate to contact me.</p>
```

Comment:
- [ ] OPTIONAL: The text here looks too congested because you are not using the font provided in the original design. Please consider using `Poppins` font family.